### PR TITLE
Scroll cards

### PR DESCRIPTION
**Pull request by RoriCremer**
For cards with more than 10 items--return all items and show all with scroll (first 10 visible without scrolling)
screen shot 2018-11-05 at 1 41 44 pm

### CODE REVIEW

**Comment by melissachang** [comment]
Is this ready for review? (The "Preview" makes me not sure.)

File: api/api.yaml [hunk 1]
```diff
@@ -49,6 +49,14 @@ paths:
           type: array
           items:
             type: string
+        - name: size
```
[the remark this answers is not in the document]
Comment (melissachang) [reply api/api.yaml]: The API server change is pretty simple, if you want to add to this PR. In [dataset_faceted_search.py](https://github.com/DataBiosphere/data-explorer/blob/master/api/data_explorer/util/elasticsearch_util.py#L258):
```
es_facet = TermsFacet(field=elasticsearch_field_name + '.keyword', size=1000)
```

**Comment by melissachang** [comment]
Making @bfcrampton reviewer since I'll be out tomorrow.

**Comment by RoriCremer** [comment]
@melissachang github closes prs automatically when you wipe commits and it's all new code

File: ui/src/components/facets/FacetCard.js [hunk 2]
```diff
@@ -8,17 +8,25 @@ import ListItemText from "@material-ui/core/ListItemText";
...
-    "&:hover": {
-      backgroundColor: "unset"
-    }
-  }
-});
+const styles = {
+  facetValueList: {
+    gridColumn: '1 / 3',
+    margin: '20px 0 0 0',
+    maxHeight: '600px',
```
Comment (melissachang) [inline ui/src/components/facets/FacetCard.js]: Now I prefer 500px

**Comment by RoriCremer** [comment]
@bfcrampton can I update and merge this?

File: ui/tests/e2e/test.js [hunk 3]
```diff
@@ -124,7 +125,7 @@ describe("End-to-end", () => {
     expect(chip.length).toBe(1);
     // Wait for the facet card to be rendered and then assert.
     await waitForFacetCard("Avuncular");
-    await assertFacet("Avuncular", "10", "HG00658 (aunt/uncle)", "1");
+    await assertFacet("Avuncular", "46", "HG00658 (aunt/uncle)", "1");
```
Comment (melissachang) [inline ui/tests/e2e/test.js]: This shouldn't change (10 -> 46).

@bfcrampton , do you mind looking at the bug in the UI tests that I mentioned earlier? I wonder if that's related. I have a meeting now, I'll be in touch after the meeting.

Comment (bfcrampton) [reply ui/tests/e2e/test.js]: Sure looking into it now.

Comment (melissachang) [reply ui/tests/e2e/test.js]: Sorry I looked into this and 46 is right, since we're returning more facet values now. Please merge.

**Pull request merged**